pkg/kev/converter/kubernetes/project_service_test.go

```diff
@@ -933,10 +936,19 @@ var _ = Describe("ProjectService", func() {
 		Context("when defined via labels", func() {
 			policy := config.DefaultRestartPolicy
 
-			BeforeEach(func() {
-				labels = composego.Labels{
-					config.LabelWorkloadRestartPolicy: policy,
+			JustBeforeEach(func() {
+				projectService.K8SConfig.Workload.RestartPolicy = policy
+				m, err := projectService.K8SConfig.ToMap()
+				Expect(err).NotTo(HaveOccurred())
+
+				svc := projectService.ServiceConfig
+				svc.Extensions = map[string]interface{}{
+					config.K8SExtensionKey: m,
 				}
+
+				projectService, err = NewProjectService(svc)
```

Comment:
These 3 lines are the setup for the following it. Handling errors is important in case the code is broken, I can remove the assertion for the restart policy is that's what you think is out of place? I don't see a point in having an IT block since it needs to be here to ensure the test is setup as expected.

Reply:
I can move the assertion down to the IT block as well